src/client/options/mod.rs

```diff
@@ -592,9 +598,8 @@ impl TlsOptions {
         if let Some(path) = self.ca_file_path {
             store
                 .add_pem_file(&mut BufReader::new(File::open(&path)?))
-                .map_err(|_| ErrorKind::ParseError {
```

Comment:
These `ParseError`'s only ever occurred when parsing TLS certificates, so I updated them to use the `TlsConfigError` variant.